src/instruments/src/EFB/Ground/Pages/Payload/Payload.tsx

```diff
@@ -23,6 +23,40 @@ import { useAppSelector } from '../../../Store/store';
 export const Payload = () => {
     const { usingMetric } = Units;
 
+    const pax: number[] = [];
+    const paxDesired: number[] = [];
+    const setPaxDesired: ((newValueOrSetter: any) => void)[] = [];
+    const activeFlags: BitFlags[] = [];
+    const setActiveFlags: ((newValueOrSetter: any) => void)[] = [];
+    const desiredFlags: BitFlags[] = [];
+    const setDesiredFlags: ((newValueOrSetter: any) => void)[] = [];
+
+    const cargo: number[] = [];
+    const cargoDesired: number[] = [];
+    const setCargoDesired: ((newValueOrSetter: any) => void)[] = [];
+
+    Loadsheet.seatMap.forEach((station) => {
+        const [stationPax] = useSimVar(`L:${station.simVar}`, 'Number', 300);
+        pax.push(stationPax);
+        const [stationPaxDesired, setStationPaxDesired] = useSimVar(`L:${station.simVar}_DESIRED`, 'Number', 300);
```

Comment:
Hooks should not be called in control structures, like loops. This can break for various reasons and cause hard-to-debug bugs in future. See the [Rules of Hooks](https://reactjs.org/docs/hooks-rules.html).